src/subdomains/core/buy-crypto/process/services/buy-crypto.service.ts

```diff
@@ -250,6 +262,71 @@ export class BuyCryptoService {
 
   // --- HELPER METHODS --- //
 
+  private async saveAmlCheck() {
+    const entities = await this.buyCryptoRepo.find({
+      where: { amlCheck: IsNull(), amlReason: IsNull(), bankTx: Not(IsNull()) },
+      relations: ['bankTx', 'buy', 'buy.user', 'buy.user.userData', 'buy.user.userData.users'],
+    });
+
+    const assetSpecifications = await this.transactionSpecificationRepo.find({ where: { system: Not('Fiat') } });
```

Comment:
Remove the repo and use the TransactionHelper. If you use the `getTxDetails` method, you will have the minVolume/minFee in input fiat.